src/core/client/admin/components/StoryInfoDrawer/StoryInfoDrawerQuery.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FunctionComponent } from "react";
+import { graphql } from "react-relay";
+
+import { QueryRenderData, QueryRenderer } from "coral-framework/lib/relay";
+import { Spinner } from "coral-ui/components/v2";
+import { QueryError } from "coral-ui/components/v3";
+
+import StoryInfoDrawerContainer from "./StoryInfoDrawerContainer";
+
+import { StoryInfoDrawerQuery as QueryTypes } from "coral-admin/__generated__/StoryInfoDrawerQuery.graphql";
+
+export interface Props {
+  storyID: string;
+  onClose: () => void;
+}
+
+const StoryInfoDrawerQuery: FunctionComponent<Props> = ({
+  storyID,
+  onClose,
+}) => {
+  return (
+    <QueryRenderer<QueryTypes>
+      query={graphql`
+        query StoryInfoDrawerQuery($storyID: ID!) {
+          story(id: $storyID) {
+            ...StoryInfoDrawerContainer_story
+          }
+        }
+      `}
```

Comment:
Seeing this query come first on the `QueryRendererer` instantiation looks really clean. I think I will start doing the same. I like how readable this is.